### ISSUE TITLE
SGP30 Sensor Not Working - Adafruit_SGP30 Lib Needs Updated

### ISSUE THREAD

**Issue by peteakalad** [arendst/Tasmota#8518]
Newly purchased SGP30 sensor is not working with current builds of Tasmota.
- Wemos / Lolin D1 Mini
- Tested on sensors bin for 8.3.1 and 7.2.0
- Generic module configuration with GPIO4 = I2C SDA & GPIO = I2C SCL
- I2CScan could see the device with standard device address 0x58
- No information shown on main tasmota page

To diagnose and confirm the sensor was not broken I tested the SGP30 example sketch in the Arduino IDE.  This worked fine.

I noticed the Adafruit_SGP30 library used by Tasmota is 1.0.3 vs 1.2.0 current version in manage libs of arduino ide.

I tested the example sketch with a downgraded version of Adafruit_SGP30 to 1.0.3.  The sketch would not work and could not find the sensor.

Having confirmed that the old library version is incompatible/broken with my sensor I compiled a sensors bin of tasmota with Adafruit_SPG30 upgraded to 1.2.0 - the sensor then started working in Tasmota.

TLDR; In summary - the current Adafruit_SGP30 lib referenced in Tasmota is broken for some versions of the SGP30 device.  The Adafruit_SGP30 library needs to be upgraded to latest 1.2.0.

SGP30 Sensor Purchased:
https://www.aliexpress.com/item/4000575428246.html
